### PR TITLE
Chapter 4

### PR DESCRIPTION
**Pull request by TuTranVan**
screenshot from 2019-02-15 09-04-35

### CODE REVIEW

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

**Comment by TuTranVan** [comment]
ready

File: app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<% provide(:title, "About") %>
+<h1><%= t "static_pages.about" %></h1>
+<p>
+  The <a href="https://www.railstutorial.org/"><em><%= t "tutorial" %></em></a>
+  is a <a href="https://www.railstutorial.org/book">book</a> and
+  <a href="http://screencasts.railstutorial.org/">screencast series</a>
+  to teach web development with
```
Comment (phamvanbk11) [inline app/views/static_pages/about.html.erb]: I18n hết nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by TuTranVan** [comment]
ready

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
 # ApplicationHelper
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title(page_title = '')
+    base_title = 'Ruby on Rails Tutorial Sample App'
```
Comment (phamvanbk11) [inline app/helpers/application_helper.rb]: dùng I18n nhé

**Comment by phamvanbk11** [comment]
commented

**Comment by TuTranVan** [comment]
ready

**Comment by phamvanbk11** [comment]
conflicted

**Comment by TuTranVan** [comment]
ready

**Comment by phamvanbk11** [comment]
merged

**Pull request merged**